packages/suite/src/views/wallet/send/components/Options/BitcoinOptions/BitcoinOptions.tsx

```diff
@@ -85,6 +89,13 @@ export const BitcoinOptions = () => {
         setDraftSaveRequest(true);
     };
 
+    const blockchain = useSelector(selectNetworkBlockchainInfo(network.symbol));
+
+    const locktime = watch('bitcoinLockTime');
+
+    const broadcastDisabled =
```

Comment:
nit: boolean variables should ideally start with `is` or `has` prefix for better readability.
```suggestion
    const isBroadcastDisabled =
```

Reply:
Do we have a consensus on this? I see both approaches in the code. I don't mind using either of them.

Reply:
very vaguely described here: https://www.notion.so/satoshilabs/Naming-6f8117ecfcab4a198faf9743467abfd8?pvs=4#8fff06b677164732b10bff0011b83024

Reply:
fixed